prisma/seeders/pcListingsSeeder.ts

```
@@ -0,0 +1,180 @@
+import { calculateWilsonScore } from '@/utils/wilson-score'
+import { ApprovalStatus, PcOs, Role, type PrismaClient } from '@orm/client'
```

ESLint

[error] `@orm/client` import should occur before import of `@/utils/wilson-score`

(import/order)